core/mantle_upgrade.go

```diff
@@ -8,26 +8,32 @@ import (
 
 var (
 	MantleMainnetUpgradeConfig = MantleUpgradeChainConfig{
-		ChainID:     params.MantleMainnetChainId,
-		BaseFeeTime: u64Ptr(0),
+		ChainID:               params.MantleMainnetChainId,
+		BaseFeeTime:           u64Ptr(0),
+		BVMETHMintUpgradeTime: u64Ptr(0),
 	}
 
 	MantleSepoliaUpgradeConfig = MantleUpgradeChainConfig{
-		ChainID:     params.MantleSepoliaChainId,
-		BaseFeeTime: u64Ptr(1_704_891_600),
+		ChainID:               params.MantleSepoliaChainId,
+		BaseFeeTime:           u64Ptr(1_704_891_600),
+		BVMETHMintUpgradeTime: u64Ptr(1_705_680_020),
```

Comment:
`BVMETHMintUpgradeTime` is not determined yet, please reset it with a future time.